presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+* Add BigQuery Connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve ORC writer performance for string columns.
+* Add BigQuery Connector.
+* Add support for improving cache affinity hit rate by preventing repartitioning when nodes go down for a limited time. This can be enabled by setting the `internal-communication.memoize-dead-notes-enabled` configuration property
```

Comment:
```suggestion
* Add support for improving cache affinity hit rate by preventing repartitioning when nodes go down for a limited time. This can be enabled by setting the `internal-communication.memoize-dead-nodes-enabled` configuration property
```